sdk/storage/tests.yml

```diff
@@ -24,5 +24,9 @@ jobs:
         AZURE_STORAGE_QUEUE_CONNECTION_STRING: $(java-storage-test-queue-connection-string)
         AZURE_STORAGE_BLOB_CONNECTION_STRING: $(java-storage-test-blob-connection-string)
         STORAGE_DATA_LAKE_ACCOUNT_NAME: $(java-storage-data-lake-account-name)
-        STORAGE_DATA_LAKE_ACCOUNT_KEY: $(java-storage-data-lake-account-key)
+        STORAGE_DATA_LAKE_ACCOUNT_KEY: $(java-storage-data-lake-account-key)[3:00 PM] Daniel Jurek
+        AZURE_STORAGE_FILE_ACCOUNT_NAME: $(java-storage-account-name)
+        AZURE_STORAGE_FILE_ACCOUNT_KEY: $(java-storage-account-key)
+        AZURE_STORAGE_QUEUE_ACCOUNT_NAME: $(java-storage-account-name)
+        AZURE_STORAGE_QUEUE_ACCOUNT_KEY: $(java-storage-account-key)
```

Comment:
Did the changes which added these remove any of the older testing environment values such as File and Queue connection string and endpoint?

Good change to make Files and Queues more common with Blobs in terms of testing configurations and setup.

Reply:
nope, I havent removed them for now since I didnt want to scope creep, I'll create an issue for this.

Reply:
https://github.com/Azure/azure-sdk-for-java/issues/6605